snappass/main.py

```diff
@@ -190,6 +198,12 @@ def show_password(password_key):
 
     return render_template('password.html', password=password)
 
+@app.errorhandler(404)
+def page_not_found(e):
+    if request.path.startswith("/snappass"):
+        return render_template('404snap.html'),404
+    else:
+        return render_template('404.html',path=request.path),404
```

Comment:
Is `path` used by the current template?

Reply:
Removed path reference (not needed).  I attempted to fix any style deviations as well.